flink-formats/flink-parquet/src/test/java/org/apache/flink/formats/parquet/ParquetColumnarRowInputFormatTest.java

```diff
@@ -126,7 +126,7 @@ class ParquetColumnarRowInputFormatTest {
                     new MapType(new IntType(), new BooleanType()),
                     RowType.of(new VarCharType(VarCharType.MAX_LENGTH), new IntType()));
 
-    @TempDir private java.io.File folder;
```

Comment:
Hello!  I refer to the two minor changes in the commit message:

1) Clean up qualified classes, and
2) Clean up @Timeout 

(Admittedly I should have used the right Case for @Timeout).

Do you think this needs to be fixed?

Reply:
Yes, it should because you're introducing a code change in this PR that you fix in the very same PR. But admittedly, we can just squash the two commits into one, anyway. So, I'm gonna go ahead with it.